package com.test6;
public class Test1 {
	public static void main(String[] args) {
//		try {
			int a = 10;
			int b = 0;
			int c = a / b;
//		} catch(ArithmeticException e) { // ArithmeticException e = new ArithmeticException();
//			System.out.println("0으로 나누셨군요.");
//			System.out.println(e.getMessage());
//		}
	}
}


